.gitignore

```diff
@@ -12,6 +12,7 @@ dist
 .browser-refresh
 integration/template.marko
 integration/browser.json
+.idea
```

Comment:
Curious, what file extension is this? And do we really need it?

Reply:
This is for IntelliJ-type editors. There's actually a lot of folks within eBay using it. It's not the first time I've seen it added (I simply moved it to be next to the one for VSCode).

Not "necessary", but is nice to have.